tests/SchemaParserTest.php

```diff
@@ -73,24 +87,29 @@ class TestEntity
     public array $hashmap = [];
 
     public TestSubEntity $class;
+
+    /**
+     * @var int|string|null
+     *
+     * @OA\Union(types={ @OA\IntegerValue(minimum=2), @OA\StringValue(minLength=10) })
```

Comment:
tohle nevim jeslti mi prijde 100% stastne - jde o to, ze me napada napr situace tato:
```
@var int[]|string[]
```
a pokud ti pak napisu
```
@OA\Union(types={ @OA\ArrayValue(minItems=2) })
```
coz je naprosto validni, tak jak to priradis?

Reply:
deje se to na zaklade pozic, takze se tim oanotuje int[]

Reply:
aha ok to me nenapadlo, prima 👍